src/Microsoft.ML.Core/Data/ReadOnlyMemoryUtils.cs

```diff
@@ -0,0 +1,399 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime.Internal.Utilities;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.ML.Runtime.Data
+{
+    public static class ReadOnlyMemoryUtils
+    {
+
+        /// <summary>
+        /// This implements IEquatable's Equals method.
```

Comment:
>This implements IEquatable's Equals method. [](start = 12, length = 43)

It couldn't possibly be implementing an interface method since this is a static method on a static class, and even if it wasn't you couldn't make `ReadOnlyMemory<char>` implement any interface anyway, since it is not a type under your control.

I *think* you meant to say it was a utility function out of which one could make an `IEquatityComparer<ReadOnlyMemory<char>>`, but it's difficult to be certain.

Reply:
Also please try to use `<see` tags.

---
In reply to: [216126127](https://github.com/dotnet/machinelearning/pull/863#discussion_r216126127) [](ancestors = 216126127)